### DISCUSSION TITLE
Practica 2 buyer persona ¿Como se hace?

### DISCUSSION THREAD

**Discussion by BenitoDelaRosaChavez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#72] [Q&A] [answered]
Hola alguien me puede ayudar en como hacer la practica 2 para crear la buyer persona, 
Gracias!

**Comment by DiegoFletes** [comment, marked as answer]
Copio y pego: Los buyer persona son representaciones de tus clientes objetivo. Crear un buyer persona implica investigar y describir los objetivos, el comportamiento, los puntos débiles y la información demográfica de tu cliente ideal.

https://es.venngage.com/blog/buyer-persona/

**Reply by DiegoFletes** [reply]
https://www.hubspot.es/make-my-persona
